core/src/main/java/tc/oc/pgm/command/CycleCommand.java

```diff
@@ -38,14 +37,14 @@ public void cycle(
     match.needModule(CycleMatchModule.class).startCountdown(duration);
   }
 
-  @Command(
-      aliases = {"recycle", "rematch"},
-      desc = "Reload (cycle to) the current map",
-      usage = "[seconds]",
-      flags = "f",
-      perms = Permissions.START)
+  @CommandMethod("recycle|rematch [duration]")
+  @CommandDescription("Reload (cycle to) the current map")
```

Comment:
Always wanted to see if we can make these translatable. Do you think it’s possible with cloud?

Reply:
the framework is flexible enough that i'm certain it can be done, but it requires using our own annotation

Reply:
We handle all translations in PGM per-sender, via adventure. We have our own systems for that here. Ideally we'd simply make the descriptions be an adventure component created from a translatable with a key specified in the `@CommandDescription` annotation, then in MinecraftHelp make sure it uses the components

Reply:
With builders you can set a component in the command meta directly (see MinecraftExtrasMetaKeys), with annotations you probably need a command builder modifier to retrieve component instances by key or something. Can either use the CommandDescription itself to store the intermediary key, or could add your own annotation and meta value